test/client.test.js

```diff
@@ -434,6 +434,42 @@ describe('ACC Client', function () {
             await client.NLWS.xtkSession.logoff();
         });
 
+        it("Should return temp group schema definition", async () => {
+            const client = await Mock.makeClient();
+            client._transport.mockReturnValueOnce(Mock.LOGON_RESPONSE);
+            await client.NLWS.xtkSession.logon();
+           
+            client._transport.mockReturnValueOnce(Mock.GET_XTK_QUERY_SCHEMA_RESPONSE);
+            client._transport.mockReturnValueOnce(Promise.resolve(`<?xml version='1.0'?>
+              <SOAP-ENV:Envelope xmlns:xsd='http://www.w3.org/2001/XMLSchema' xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xmlns:ns='urn:xtk:queryDef' xmlns:SOAP-ENV='http://schemas.xmlsoap.org/soap/envelope/'>
+              <SOAP-ENV:Body>
+              <ExecuteQueryResponse xmlns='urn:xtk:queryDef' SOAP-ENV:encodingStyle='http://schemas.xmlsoap.org/soap/encoding/'>
+                  <pdomOutput xsi:type='ns:Element' SOAP-ENV:encodingStyle='http://xml.apache.org/xml-soap/literalxml'>
+                  <group expirationDate="" folder-id="1199" id="2200" label="testlist" name="LST260" schema="nms:recipient" type="1">
+                    <extension label="email is not empty" mappingType="sql" name="query" namespace="temp">
+                      <element advanced="false" dataSource="nms:extAccount:ffda" label="email is not empty" name="query" pkSequence="" sqltable="grp2200" unbound="false">
+                        <compute-string expr=""/>
+                        <key internal="true" name="internal">
+                          <keyfield xpath="@id"/>
+                        </key>
+                        <attribute advanced="false" belongsTo="@id" label="Primary key" length="0" name="id" notNull="false" sql="true" sqlname="uId" type="uuid" xml="false"/>
+                        <element advanced="false" externalJoin="true" label="Targeting dimension" name="target" revLink="" target="nms:recipient" type="link" unbound="false">
+                          <join xpath-dst="@id" xpath-src="@id"/>
+                        </element>
+                      </element>
+                    </extension>
+                    <desc><![CDATA[]]></desc>
+                    <folder _cs="Lists" fullName="/Profiles and Targets/Lists/" id="1199"/>
+                  </group>
+                  </pdomOutput></ExecuteQueryResponse>
+              </SOAP-ENV:Body>
+              </SOAP-ENV:Envelope>`));
+          
+            var schema = await client.getSchema("temp:group:2200");
+            expect(schema["namespace"]).toBe("temp");
+            expect(schema["name"]).toBe("query");
```

Comment:
Can you add a test (in application.test.js) which uses the getSchema function of the application object and also goes a little bit deeper in the assertions, i.e. which checks that the <key>, <attribute>, etc. are properly returned